sdk/nodejs/appservice/zMixins.ts

```diff
@@ -89,6 +89,13 @@ export type CallbackFunctionAppArgs<C extends Context<R>, E, R extends Result> =
      */
     callbackFactory?: CallbackFactory<C, E, R>;
 
+    /**
+     * The bindings to be used for the function.
+     * See https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings 
+     * for more details
+     */
+    bindings?: pulumi.Input<azurefunctions.BindingDefinition[]>;

```

Comment:
ideall, this should be written as `pulumi.Input<pulumi.Input<azurefunctions.BindingDefinition>[]>`;

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b